src/functions/loadData.ts

```diff
@@ -1,16 +1,28 @@
 import SimpleData from "../class/SimpleData.js";
 import checkEnvironment from "../helpers/checkEnvironment.js";
 import log from "../helpers/log.js";
-import { Options, defaultOptions } from "../types/SimpleData.types.js"
 import showTable from "../methods/showTable.js";
 import getExtension from "../helpers/getExtension.js";
 
 
-export default async function loadData(path: string, options: Options) {
-
-    options = {
-        ...defaultOptions,
-        ...options
+export default async function loadData({
+    path, 
+    verbose = false, 
```

Comment:
verbose is probably not very clear for new users. Maybe something more explicit?